src/OrchardCore.Modules/OrchardCore.Resources/package.json

```diff
@@ -13,7 +13,7 @@
     "jquery-resizable-dom": "0.35.0",
     "jquery.easing": "1.4.1",
     "js-cookie": "3.0.1",
-    "monaco-editor": "^0.35.0",
+    "monaco-editor": "^0.36.1",
```

Comment:
I get this in the dev console under https://localhost:44300/Admin/ContentTypes/Article/ContentParts/HtmlBodyPart/Edit when set up with the Blog recipe:

```
DevTools failed to load source map: Could not load content for https://localhost:44300/OrchardCore.Resources/Scripts/min-maps/vs/loader.js.map: HTTP error: status code 404, net::ERR_HTTP_RESPONSE_CODE_FAILURE
DevTools failed to load source map: Could not load content for chrome-extension://gpaiobkfhnonedkhhfjpmhdalgeoebfa/editor/config.js.map: System error: net::ERR_BLOCKED_BY_CLIENT
DevTools failed to load source map: Could not load content for chrome-extension://gpaiobkfhnonedkhhfjpmhdalgeoebfa/editor/content.js.map: System error: net::ERR_BLOCKED_BY_CLIENT
DevTools failed to load source map: Could not load content for https://localhost:44300/OrchardCore.Resources/Scripts/min-maps/vs/editor/editor.main.js.map: HTTP error: status code 404, net::ERR_HTTP_RESPONSE_CODE_FAILURE
DevTools failed to load source map: Could not load content for https://localhost:44300/OrchardCore.Resources/Scripts/min-maps/vs/editor/editor.main.nls.js.map: HTTP error: status code 404, net::ERR_HTTP_RESPONSE_CODE_FAILURE
DevTools failed to load source map: Could not load content for https://localhost:44300/OrchardCore.Resources/Scripts/min-maps/vs/base/worker/workerMain.js.map: Fetch through target failed: Target not supported; Fallback: HTTP error: status code 404, net::ERR_HTTP_RESPONSE_CODE_FAILURE
DevTools failed to load source map: Could not load content for https://localhost:44300/OrchardCore.Resources/Scripts/min-maps/vs/base/worker/workerMain.js.map: Fetch through target failed: Target not supported; Fallback: HTTP error: status code 404, net::ERR_HTTP_RESPONSE_CODE_FAILURE
DevTools failed to load source map: Could not load content for https://localhost:44300/Admin/min-maps/vs/base/common/worker/simpleWorker.nls.js.map: Fetch through target failed: Target not supported; Fallback: HTTP error: status code 404, net::ERR_HTTP_RESPONSE_CODE_FAILURE
DevTools failed to load source map: Could not load content for https://localhost:44300/Admin/min-maps/vs/base/common/worker/simpleWorker.nls.js.map: Fetch through target failed: Target not supported; Fallback: HTTP error: status code 404, net::ERR_HTTP_RESPONSE_CODE_FAILURE
jsonWorker.js:34 Not allowed to load local resource: file:///OrchardCore.Resources/Scripts/monaco/IStandaloneEditorConstructionOptions.json
zr @ jsonWorker.js:34
t.loadSchema @ jsonWorker.js:33
t.getUnresolvedSchema @ jsonWorker.js:33
t.getResolvedSchema @ jsonWorker.js:33
t.getSchemaForResource @ jsonWorker.js:33
t.findDocumentColors @ jsonWorker.js:33
findDocumentColors @ jsonWorker.js:34
fmr @ workerMain.js:22
d @ workerMain.js:22
handleMessage @ workerMain.js:22
k @ workerMain.js:22
h @ workerMain.js:22
handleMessage @ workerMain.js:22
onmessage @ workerMain.js:22
self.onmessage @ workerMain.js:11
```

No such issue in the Template editor.